ras_party/controllers/queries.py

```diff
@@ -173,23 +173,33 @@ def search_businesses(search_query, session):
     :param search_query: the search query
     :return: list of businesses
     """
```

Comment:
May be worth some more explanation of what sort of things this function is expecting (what do we mean by "search_query"?) and the logic flow

Reply:
Done